cmd/gpctl/cmd/status.go

```diff
@@ -90,18 +73,54 @@ func statusEntrypoint(ctx context.Context, cmd *cobra.Command, args []string) er
 		return allStatuses[i].iface < allStatuses[j].iface
 	})
 
-	for _, st := range allStatuses {
-		status.Line(st.iface)
+	bold := color.New(color.Bold, color.FgWhite)
+	boldRed := color.New(color.Bold, color.FgRed)
+
+	table := tablewriter.CreateTable()
+	table.UTF8Box()
+	table.AddTitle(bold.Sprint("Interface Statuses"))
 
+	table.AddRow("", "total", "", "total", "", "")
+	table.AddRow("iface",
+		"received", "+ received",
+		"processed", "+ processed",
+		"dropped",
+	)
+	table.AddSeparator()
+
+	for _, st := range allStatuses {
 		ifaceStatus := st.status
 
+		runtimeTotalReceived += int64(ifaceStatus.ReceivedTotal)
+		runtimeTotalProcessed += int64(ifaceStatus.ProcessedTotal)
+
+		totalProcessed += int64(ifaceStatus.Processed)
 		totalReceived += int64(ifaceStatus.Received)
 		totalDropped += int64(ifaceStatus.Dropped)
 		totalActive++
 
-		printStats(ifaceStatus)
+		dropped := fmt.Sprint(ifaceStatus.Dropped)
+		if ifaceStatus.Dropped > 0 {
+			dropped = boldRed.Sprint(ifaceStatus.Dropped)
+		}
+
+		table.AddRow(st.iface,
+			formatting.Countable(ifaceStatus.ReceivedTotal), formatting.Countable(ifaceStatus.Received),
+			formatting.Countable(ifaceStatus.ProcessedTotal), formatting.Countable(ifaceStatus.Processed),
+			dropped,
+		)
 	}
 
+	// set alignment before rendering
+	table.SetAlign(tablewriter.AlignLeft, 1)
+	table.SetAlign(tablewriter.AlignRight, 2)
+	table.SetAlign(tablewriter.AlignRight, 3)
+	table.SetAlign(tablewriter.AlignRight, 4)
+	table.SetAlign(tablewriter.AlignRight, 5)
+	table.SetAlign(tablewriter.AlignRight, 6)
+
+	fmt.Println(table.Render())
```

Comment:
It's ok that this whole thing lives in memory, given how small the tables are. Wouldn't do this for the `goquery` output, don't worry.